backend/post-service/src/main/java/com/speakapp/postservice/dtos/LatestUserPostsPageGetDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.speakapp.postservice.dtos;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class LatestUserPostsPageGetDTO {
```

Comment:
Tu pytałeś mnie o nazwę czy git, a nie lepiej po prostu "PostPageGetDTO"?  W zasadzie to nie ma w tej klasie info o userze itp. i słusznie. Taka ogólna klasa przyda się przy innych endpointach i będzie identyczna.